google-cloud-pubsub/src/test/java/com/google/cloud/pubsub/spi/v1/SubscriberImplTest.java

```diff
@@ -322,6 +322,57 @@ public ModifyAckDeadline apply(String ack) {
   }
 
   @Test
+  public void testModifyAckDeadline_defaultMaxExtensionPeriod() throws Exception {
+    Subscriber subscriber =
+        startSubscriber(
+            getTestSubscriberBuilder(testReceiver)
+                .setAckExpirationPadding(Duration.standardSeconds(1)));
+    // Send messages to be acked
+    List<String> testAckIdsBatch = ImmutableList.of("A", "B", "C");
+    testReceiver.setExplicitAck(true);
+    sendMessages(testAckIdsBatch);
+
+    // Trigger modify ack deadline sending - 10s initial stream ack deadline - 1 padding
+    fakeExecutor.advanceTime(Duration.standardSeconds(9));
+
+    assertEquivalentWithTransformation(
+        testAckIdsBatch,
+        fakeSubscriberServiceImpl.waitAndConsumeModifyAckDeadlines(3),
+        new Function<String, ModifyAckDeadline>() {
+          @Override
+          public ModifyAckDeadline apply(String ack) {
+            return new ModifyAckDeadline(ack, 2); // 2 seconds is the initial mod ack deadline
+          }
+        });
+
+    int timeIncrement = 4; // Second time increment
+
+    while (fakeExecutor.getClock().millisTime() + (timeIncrement * 1000) < 1000 * 60 * 60) {
```

Comment:
make a variable/constant out of the numerical expression to give it meaning (I can't tell at a glance how much time it is)